scrapers/Karups.yml

```diff
@@ -0,0 +1,26 @@
+name: "Karups"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.karups.com/video/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $details: //div[@class="content-information"]
```

Comment:
in some exclusive scenes ( `https://www.karups.com/video/avi-love-the-gardener-s-hose-50693.html` , `https://www.karups.com/video/lexi-29899.html` ) there is no div with "content-information"

Given that `class="date"` and `class="models"` seem unique enough the whole `$details` can be removed from `common` and the date and performers selectors